src/App.tsx

```diff
@@ -1,61 +1,132 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types';
+
+function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+export const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+function getNewTodoId(todos: Todo[]) {
+  const maxId = Math.max(
+    ...todos.map(todo => todo.id),
+  );
+
+  return maxId + 1;
+}
+
+const symbolsExsception = /[^a-zA-Zа-яА-Я0-9\s]/g;
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [count, setCount] = useState(getNewTodoId(initialTodos));
```

Comment:
Why do you have this state? 
You may calculate it when needed

Reply:
Ok - I see now - you have random id's on Todos at the start

Your solution makes sense then - please, ignore my previous comment